packages/utils/index.js

```diff
@@ -26,4 +28,10 @@ export class FabricWebComponent extends HTMLElement {
   }
 }
 
+export class FabricElement extends LitElement {
+  createRenderRoot() {
+    return this;
+  }
+}
+
```

Comment:
very simple base class that disables Shadow DOM. Is there much point in using this?